spec/models/user_profile_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe UserProfile, type: :model do
+
+  describe "relationships" do
+    it { should belong_to :user }
+  end
+
+  describe "validations" do
+    it { should validate_presence_of :first_name }
+    it { should validate_presence_of :last_name }
+    it { should validate_presence_of :program }
+    it { should validate_presence_of :current_mod }
+    it { should validate_presence_of :starting_cohort }
+    it { should validate_presence_of :current_cohort }
+    it { should define_enum_for :status }
```

Comment:
Very cool, I have not seen this syntax for enums in shouldamatchers before. Killing it!